install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -252,12 +252,20 @@ spec:
             type: 'spc_t'
           capabilities:
             add:
+              # Use to set socket permission
+              - CHOWN
               # Used to terminate envoy child process
               - KILL
               # Used since cilium modifies routing tables, etc...
               - NET_ADMIN
+              # Used since cilium creates raw sockets, etc...
+              - NET_RAW
               # Used since cilium monitor uses mmap
               - IPC_LOCK
+              # Allow to execute program that changes GID
+              - SETGID
+              # Allow to execute program that changes UID
+              - SETUID
```

Comment:
do you know which error happens if we don't have these two caps ? I tried locally without these two, but unable to find any error. Thanks.

Reply:
This is cause by testcase
https://github.com/cilium/cilium/blob/dfa6b157e8e9484c65fd938b2b45a4f5f50c61f9/test/k8s/datapath_configuration.go#L923

But I think these are common capabilities to execute system tools require to change process UID/GID.
There are other method:
1. patch DaemonSet to add these capabilities after Cilium is deployed for this testcase
2. introduce new options (plural because there are many containers that config capabilities) in helm chart allowing user to add extra capabilities for container

Reply:
If these caps are only used in the tests, we should not have them in default installation option. If I remember correctly, there was a security concern about allowing installing more packages. Related to https://github.com/cilium/cilium/issues/8761.

In short, I personally prefer to have addition helm attribute like you mentioned above (e.g. securityContext.extraCapabilities). This can be just applied for cilium agent main container for simplicity. 

/cc maintainers @aanm @joestringer

Reply:
Ok, I add it to helm values